scripts/test-data/train-profile-4xa10.yaml

```diff
@@ -0,0 +1,21 @@
+additional_args:
+  warmup_steps: 10
+  learning_rate: 2e-6
+  lora_dropout: 0.1
+  lora_alpha: 32
+  deepspeed_cpu_offload_optimizer_pin_memory: false
+  deepspeed_cpu_oddload_optimizer_ratio: 1
+ckpt_output_dir: checkpoints
+data_output_dir: train-output
+data_path: ./taxonomy_data
+deepspeed_cpu_offload_optimizer: true
+effective_batch_size: 32
+lora_quantize_dtype: null
+lora_rank: 0
+max_batch_len: 10000
+max_seq_len: 4096
+model_path: instructlab/granite-7b-lab
+num_epochs: 1
+save_samples: 0
+is_padding_free: true
+nproc_per_node: 4
```

Comment:
nit: can we rename this file to `train-profile-a10x4.yaml` so it matches the scheme of the workflow file?